client/components/Timer/timer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+
+type TimerProps = {
+  time: number;
```

Comment:
Proponuje zamiast operować na abstrakcyjnym pojęciu jakim jest `Time` zamienić to na `seconds` albo `timeInSeconds`

Reply:
ok :)